jekyll/_cci2/server/installation/migrate-from-server-2-to-server-4.adoc

```diff
@@ -0,0 +1,109 @@
+---
+contentTags: 
+  platform:
+  - Server v4.0
+  - Server Admin
+---
+= CircleCI Server v2.19 to v4.0 Migration
+:page-layout: classic-docs
+:page-liquid:
+:page-description: Learn now to migrate from CircleCI server 2.19.x to 4.0.
+:icons: font
+:toc: macro
+:toc-title:
+
+Migrating from 2.19.x to 4.0 requires you to back up your 2.19 instance data (Mongo, Postgres, and Vault) and then restore that data in a waiting Server 4.0 instance. If you experience problems, you can fall back to your 2.19 instance. Migration requires an already operating Server 4.0 installation.
+
+Depending on the size of your data stores, the migration can take anywhere from a few minutes to a few hours. We recommend using a staging environment before completing this process in a production environment. This not only allows you to gain a better understanding of the migration process, but also gives you an idea of how long the migration will take to complete.
+
+toc::[]
+
+## Prerequisites
+
+. Your current CircleCI Server installation is 2.19.
+. You have taken a backup of the 2.19 instance. If you are using external datastores, they need to be backed up separately.
+. You have a new CircleCI Server 4.0 xref:phase-1-prerequisites.adoc[installation].
+. You have successfully run https://support.circleci.com/hc/en-us/articles/360011235534-Using-realitycheck-to-validate-your-CircleCI-installation[realitycheck] with contexts before starting.
```

Comment:
Would it be ok to link here: https://circleci.com/docs/server/installation/phase-3-execution-environments/#nomad-clients-validation instead of that support article for info on realitycheck?

Reply:
maybe linking to the [realitycheck](https://github.com/circleci/realitycheck) repo may be best here since they would have gone through nomad validation when setting up their 4.0 instance